packages/vats/test/upgrading/test-upgrade-vats.js

```diff
@@ -0,0 +1,56 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { buildVatController } from '@agoric/swingset-vat';
+
+/** @type {import('ava').TestFn<Awaited<ReturnType<typeof makeTestContext>>>} */
+const test = anyTest;
+
+/**
+ * WARNING: uses ambient authority
```

Comment:
what is the importance of this warning? what risk does it pose to the interpretation or execution of this test? I get if you want to `xxx` to say this is not ocap discipline but I don't see what merits WARNING.